src/components/MapInput/index.js

```diff
@@ -47,15 +47,17 @@ const StyledAutosuggest = styled(PDOKAutoSuggest)`
   }
 `;
 
-const MapInput = ({ className, value, onChange, mapOptions, ...otherProps }) => {
+const MapInput = ({ className, value, onChange, mapOptions, events }) => {
   const { state, dispatch } = useContext(MapContext);
   const [map, setMap] = useState();
   const [marker, setMarker] = useState();
   const { location, addressText: addressValue } = state;
-  const hasLocation = location && location?.lat !== 0 && location.lng !== 0;
+  const hasLocation = Boolean(location) && location?.lat !== 0 && location?.lng !== 0;
 
   const clickFunc = useCallback(
     async event => {
+      map.flyTo(event.latlng);
```

Comment:
Is the intended functionality to center the map on the clicked location? The UX experience is not that pretty when the user will click on several locations after each other.

Reply:
I'll let our UX guy judge the change in behaviour.